L1Trigger/Phase2L1ParticleFlow/src/CaloClusterer.cc

```diff
@@ -167,9 +167,15 @@ l1tpf_calo::SingleCaloClusterer::SingleCaloClusterer(const edm::ParameterSet &ps
     : grid_(getGrid(pset.getParameter<std::string>("grid"))),
       rawet_(*grid_),
       unclustered_(*grid_),
+      eta_center_(*grid_),
+      phi_center_(*grid_),
       precluster_(*grid_),
       clusterIndex_(*grid_),
       cellKey_(*grid_),
+      preciseEtaPhi_(pset.existsAs<bool>("usePreciseEtaPhi") ? pset.getParameter<bool>("usePreciseEtaPhi") : false),
```

Comment:
Would it be feasible to rely on `fillDescriptions()` instead of calling `existsAs()`?

Reply:
In theory, yes. But given the complexity of the configuration (L1Trigger/Phase2L1ParticleFlow/python/l1tPFClustersFromCombinedCalo_cfi.py) this is probably best left to the original developers to sort out what an appropriate default is for the individual producers that use this utility, and all the other defaults for the (2?) different producers dependent on it.

Reply:
it would be possible, but in the near future this class is going do be removed in favour of the new GCT emulator of PF clustering ( https://github.com/cms-l1t-offline/cmssw/pull/1095 ), so it wouldn't be the best investment in terms of effort.